### PR TITLE
Add makeup-screenreader-trap definitions

### PR DESCRIPTION
**Pull request by darkwebdev**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/makeup-screenreader-trap/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Type definitions for makeup-screenreader-trap 0.0
+// Project: https://github.com/makeup-js/makeup-screenreader-trap#readme
+// Definitions by: Timur Manyanov <https://github.com/darkwebdev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+export function trap(el: any): any;
```
Comment (DanielRosenwasser) [inline types/makeup-screenreader-trap/index.d.ts]: Should this just be `el: HTMLElement`?

File: types/makeup-screenreader-trap/makeup-screenreader-trap-tests.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { trap, untrap } from 'makeup-screenreader-trap';
+
+const widgetEl: HTMLElement | null = document.querySelector('.expander');
+
+trap(widgetEl);
```
Comment (DanielRosenwasser) [inline types/makeup-screenreader-trap/makeup-screenreader-trap-tests.ts]: It doesn't look like the library actually accepts `null`.

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

File: types/makeup-screenreader-trap/index.d.ts (hunk 1 above)
Comment (darkwebdev) [reply types/makeup-screenreader-trap/index.d.ts]: Thanks, you're right, fixed.

File: types/makeup-screenreader-trap/makeup-screenreader-trap-tests.ts (hunk 2 above)
Comment (darkwebdev) [reply types/makeup-screenreader-trap/makeup-screenreader-trap-tests.ts]: Fixed.

**Comment by DanielRosenwasser** [comment]
Thanks!

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**